sdk/search/search-documents/CHANGELOG.md

```diff
@@ -1,15 +1,19 @@
 # Release History
 
-## 11.3.0-beta.1 (Unreleased)
+## 11.3.0-beta.1 (2021-07-07)
 
-### Features Added
 - With the dropping of support for Node.js versions that are no longer in LTS, the dependency on `@types/node` has been updated to version 12. Read our [support policy](https://github.com/Azure/azure-sdk-for-js/blob/main/SUPPORT.md) for more details.
 
-### Breaking Changes
+- Regenerated the search SDK with the latest swaggers that includes the following changes:
 
-### Key Bugs Fixed
-
-### Fixed
+  - Support for `TokenCredential` has been added.
```

Comment:
Can we re-word this to convey that authentication via AAD is now supported?

Reply:
Done